### ISSUE TITLE
graph view ignores link alias

### ISSUE THREAD

**Issue by 7flash** [zsviczian/excalibrain#94]
![image](https://user-images.githubusercontent.com/4569866/190201388-ce6aa50e-5a9e-419c-aa42-90f92a0e59ac.png)

Expected to see a child node with label "second", not "third-document"

**Comment by zsviczian** [comment]
Yes, I haven't implemented this feature yet. It is also not so trivial to implement given the data model I've built for ExcaliBrain, but it is on my list of things I'd like to do one day.

**Comment by 7flash** [comment]
https://github.com/excalidraw/excalidraw/issues/3276

**Comment by 7flash** [comment]
What's your workflow while navigating large documents currently?

Minimap with clickable links at relative positions would be perfect, but perhaps saving links to markdown already might work?
